### PR TITLE
update for latest template

### PR DESCRIPTION
**Pull request by NeoZhangJianyu**
# Existing Sample Changes
## Description

Update for the latest template.

Fixes Issue# https://jira.devtools.intel.com/browse/ONSAM-1877

## External Dependencies

List any external dependencies created as a result of this change.

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode
- [ ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used

**_Delete this line and everything below if this is not a PR for a new code sample_**

**_Delete this line and all above it if this PR is for a new code sample_**
# Adding a New Sample(s)

## Description

Please include a description of the sample

## Checklist
Administrative
- [ ] Review sample design with the appropriate [Domain Expert](https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts): <insert Name Here>
- [ ] If you have any new dependencies/binaries, inform the oneAPI Code Samples Project Manager

Code Development
- [ ] Implement coding guidelines and ensure code quality. [see wiki for details](https://github.com/oneapi-src/oneAPI-samples/wiki/General-Code-Guidelines)
- [ ] Adhere to readme template 
- [ ] Enforce format via clang-format config file
- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com

Security and Legal
- [ ] OSPDT Approval (see Project Manager for assistance)
- [ ] Compile using the following compiler flags and fix any warnings, the falgs are: "/Wall -Wformat-security -Werror=format-security"
- [ ] Bandit Scans (Python only)
- [ ] Virus scan

Review
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth(@tomlenth) and/or Project Manager
- [ ] Tested using Dev Cloud when applicable

### CODE REVIEW

**Comment by NeoZhangJianyu** [comment]
merge to wrong branch.

**Comment by NeoZhangJianyu** [comment]
Reopen.

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 1]
```diff
@@ -153,11 +153,10 @@ Create a new kernel for the Jupyter notebook based on your activated conda envir
...
 Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
 * [AI Tools Offline Installer (Validated)](#ai-tools-offline-installer-validated)
-* [Conda/PIP](#condapip) 
-* [Docker](#docker)
+* [Conda/PIP](#setup-conda-environment)
 
-### AI Tools Offline Installer (Validated)  
-1. If you have not already done so, activate the AI Tools bundle base environment. 
+### AI Tools Offline Installer (Validated)
+1. If you have not already done so, activate the AI Tools bundle base environment.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:159]: Suggested content for AI Tools Offline Installer section:
 
**1. Register Conda kernel to Jupyter Notebook kernel**

If the default path is used during the installation of AI Tools:
```
$HOME/intel/oneapi/intelpython/envs/tensorflow-gpu/bin/python -m ipykernel install --user --name=tensorflow-gpu
```
If a non-default path is used:
```
<custom_path>/bin/python -m ipykernel install --user --name=tensorflow-gpu
```
**2. Launch Jupyter Notebook** 

```
jupyter notebook --ip=0.0.0.0
```
**3. Follow the instructions to open the URL with the token in your browser**

**4. Select the Notebook**

```
inc_sample_tensorflow.ipynb
```
**5. Change the kernel to `tensorflow-gpu`**
  
**6. Run every cell in the Notebook in sequence**

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 2]
```diff
@@ -180,7 +179,8 @@ source <custom_path>/bin/activate
    ```
 2. Change to the sample directory.
 
-### Run the Notebook
+### Launch Jupyter Notebook
+
 1. Launch Jupyter Notebook.
    ```
    jupyter notebook --ip=0.0.0.0
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:186]: Suggested content for Conda/PIP section:
 
### Conda/PIP
> **Note**: Before running the instructions below, make sure your Conda/Python environment with AI Tools installed is activated

**1. Register Conda/Python kernel to Jupyter Notebook kernel** 

For Conda:
```
<CONDA_PATH_TO_ENV>/bin/python -m ipykernel install --user --name=<your-env-name>
```
To know <CONDA_PATH_TO_ENV>, run `conda env list` and find your Conda environment path.

For PIP:
```
python -m ipykernel install --user --name=<your-env-name>
```
**2. Launch Jupyter Notebook**
 
```
jupyter notebook --ip=0.0.0.0
```
**3. Follow the instructions to open the URL with the token in your browser**

**4. Select the Notebook**

```
inc_sample_tensorflow.ipynb
```
**5. Change the kernel to `<your-env-name>`**


**6. Run every cell in the Notebook in sequence**

Comment (Marina-Zubova) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:186]: Suggested content for Docker section:

### Docker
AI Tools Docker images already have Get Started samples pre-installed. Refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.

**Review by Marina-Zubova — CHANGES_REQUESTED** [review]
@NeoZhangJianyu , thank you for your updates! There are still changes needed to be made on this README to match the template. I left my suggestions, please let me know if you have any questions

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 3]
```diff
@@ -2,28 +2,28 @@
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:2]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 4]
```diff
@@ -2,28 +2,28 @@
 
 This sample demonstrates using the Intel® Neural Compressor, which is part of the Intel® AI Tools with the with Intel® Optimizations for TensorFlow* to speed up inference by simplifying the process of converting the FP32 model to INT8/BF16.
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:3]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 5]
```diff
@@ -2,28 +2,28 @@
...
-| Optimized for                     | Description
-|:---                               |:---
-| OS                                | Ubuntu* 20.04 (or newer) <br> Windows 11, 10*
-| Hardware                          | Intel® Core™ Gen10 Processor <br> Intel® Xeon® Scalable Performance processors
-| Software                          | Intel® Neural Compressor, Intel Optimization for TensorFlow
+| Optimized for                     | Description |
+|:---                               |:---         |
+| OS                                | Ubuntu* 20.04 (or newer) <br> Windows 11, 10* |
+| Hardware                          | Intel® Core™ Gen10 Processor <br> Intel® Xeon® Scalable Performance processors |
+| Software                          | Intel® Neural Compressor, Intel Optimization for TensorFlow |
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:27]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 6]
```diff
@@ -74,7 +74,7 @@ The sample demonstrates how to:
 ## Environment Setup
 If you have already set up the PIP or Conda environment and installed AI Tools go directly to Run the Notebook.
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:76]: I will update the chapter structure for above style.
The requirements.txt is used by PIP.
But in this sample, we use conda. So no need to create and use requirements.txt file.

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 7]
```diff
@@ -74,7 +74,7 @@ The sample demonstrates how to:
...
 
-### On Linux* 
+### On Linux*
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:77]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 8]
```diff
@@ -153,11 +153,10 @@ Create a new kernel for the Jupyter notebook based on your activated conda envir
 > Linux*:
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:153]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 1 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:159]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 9]
```diff
@@ -153,11 +153,10 @@ Create a new kernel for the Jupyter notebook based on your activated conda envir
...
 Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
 * [AI Tools Offline Installer (Validated)](#ai-tools-offline-installer-validated)
-* [Conda/PIP](#condapip) 
-* [Docker](#docker)
+* [Conda/PIP](#setup-conda-environment)
```
[the remark this answers is not in the document]
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md]: yes, add them.

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 10]
```diff
@@ -23,7 +23,8 @@ You will learn how to train a CNN model with Keras and TensorFlow*, use Intel®
 |:---                               |:---         |
 | OS                                | Ubuntu* 20.04 (or newer) <br> Windows 11, 10* |
 | Hardware                          | Intel® Core™ Gen10 Processor <br> Intel® Xeon® Scalable Performance processors |
-| Software                          | Intel® Neural Compressor, Intel Optimization for TensorFlow |
+| Software                          | Intel® Neural Compressor, Intel® Extension for TensorFlow* |
+> **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
 
 ### Intel® Neural Compressor and Sample Code Versions
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:30]: It seems like Intel® AI Tools has been renamed to just AI Tools, we need to update this everywhere in the README

Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:30]: Section 'For Local Development Environments' - we already cover all the needed instructions in Env Setup and Run a sample sections, so this section is redundant and should be removed

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 11]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
-### On Linux*
+### 1. Get AI Tools
+
+Required AI Tools: Intel® Neural Compressor, Intel® Extension for TensorFlow* (GPU)
 
-#### Setup Conda Environment
+If you have not already, select and install these Tools via [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.
 
-You can list the available conda environments using a command similar to the following.
+Note: If Docker option is chosen in AI Tools Selector, refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:87]: fixing formatting for a note
```suggestion
>**Note**: If Docker option is chosen in AI Tools Selector, refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 12]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
 ## Environment Setup
+
+You will need to download and install the following toolkits, tools, and components to use the sample.
+
 If you have already set up the PIP or Conda environment and installed AI Tools go directly to Run the Notebook.
 
-### On Linux*
+### 1. Get AI Tools
+
+Required AI Tools: Intel® Neural Compressor, Intel® Extension for TensorFlow* (GPU)
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:83]: seems like this sample is a CPU sample?
```suggestion
Required AI Tools: Intel® Neural Compressor, Intel® Extension for TensorFlow* (CPU)
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 13]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow
+```
+
+### 5. Install dependencies
+
+Install for Jupyter Notebook:
+
+```
+conda install jupyter runipy notebook -y
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:121]: it's recommended to but dependent packages into requirements.txt file and install everything with `pip install -r requirements.txt`.
In any case, we need to make sure we list all the needed packages, right now I see that ipykernel and matplotlib packages are missing in the command

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 14]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
+* [Docker](#docker)
+
 
 ### AI Tools Offline Installer (Validated)
-1. If you have not already done so, activate the AI Tools bundle base environment.
-If you used the default location to install AI Tools, open a terminal and type the following
+
+#### 1. Register Conda kernel to Jupyter Notebook kernel
+
+* Note: If you have done this step before, skip it.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:142]: note formatting
```suggestion
> **Note**: If you have done this step before, skip it.
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 15]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
 ```
 
-### Active Conda Environment
+#### 2. Launch Jupyter Notebook
 
-1. Ensure you activate the conda environment.
-   ```
-   source /opt/intel/oneapi/setvars.sh
-   conda activate tensorflow
+- Alternative 1: Launch Jupyter Notebook.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:158]:
```suggestion
- Option A: Launch Jupyter Notebook.
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 16]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
    ```
-   or
-   ```
-   conda activate usr_tensorflow
+   jupyter notebook --ip=0.0.0.0
    ```
-2. Change to the sample directory.
 
-### Launch Jupyter Notebook
+- Alternative 2: You can launch Jupyter Notebook by running the script located in the sample code directory.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:163]:
```suggestion
- Option B: You can launch Jupyter Notebook by running the script located in the sample code directory.
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 17]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
+#### 3. Follow the instructions to open the URL with the token in your browser
+
+#### 4. Select the Notebook
+
+```
+inc_sample_tensorflow.ipynb
+```
+
+
+#### 5. Change the kernel to **tensorflow**
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:198]: fixing formatting
```suggestion
#### 5. Change the kernel to `tensorflow`
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 18]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
+
+#### 6. Run every cell in the Notebook in sequence
+
+
+
+### Conda/PIP
+
+- Note: Before running the instructions below, make sure your Conda/Python environment with AI Tools installed is activated
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:206]:
```suggestion
> **Note**: Before running the instructions below, make sure your Conda/Python environment with AI Tools installed is activated
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 19]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
+
+
+#### 1. Register Conda/Python kernel to Jupyter Notebook kernel
+
+* Note: If you have done this step before, skip it.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:211]:
```suggestion
> **Note**: If you have done this step before, skip it.
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 20]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
+For PIP:
+
+```
+python -m ipykernel install --user --name=tensorflow
+```
+
+#### 2. Launch Jupyter Notebook
+
+
+- Alternative 1: Launch Jupyter Notebook.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:229]:
```suggestion
- Option A: Launch Jupyter Notebook.
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 21]
```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
...
    ```
    jupyter notebook --ip=0.0.0.0
    ```
-2. Alternatively, you can launch Jupyter Notebook by running the script located in the sample code directory.
+
+- Alternative 2: You can launch Jupyter Notebook by running the script located in the sample code directory.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:234]:
```suggestion
- Option B: You can launch Jupyter Notebook by running the script located in the sample code directory.
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 22]
```diff
@@ -209,15 +256,20 @@ source <custom_path>/bin/activate
...
-   ```
-   inc_sample_tensorflow.ipynb
-   ```
-4. Change the kernel to **user_tensorflow**.
-5. Run every cell in the Notebook in sequence.
+#### 3. Follow the instructions to open the URL with the token in your browser
+
+#### 4. Select the Notebook
+
+inc_sample_tensorflow.ipynb
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:264]: formatting
```
inc_sample_tensorflow.ipynb
```

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md [hunk 23]
```diff
@@ -209,15 +256,20 @@ source <custom_path>/bin/activate
...
+
+#### 5. Change the kernel to **tensorflow**
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:266]:
```suggestion
#### 5. Change the kernel to `tensorflow`
```

**Review by Marina-Zubova — CHANGES_REQUESTED** [review]
the README has been improved a lot, there are just some minor changes needed, I left my comments

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 10 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:30]: yes, update

Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:30]: yes, remove it.

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 11 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:87]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 12 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:83]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 13 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:121]: yes, create requirements.txt

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 14 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:142]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 15 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:158]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 17 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:198]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 18 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:206]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 19 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:211]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 20 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:229]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 21 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:234]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 22 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:264]: yes, update

File: AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md (hunk 23 above)
Comment (NeoZhangJianyu) [reply AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md:266]: yes, update

**Comment by jimmytwei** [comment]
@NeoZhangJianyu Do the other files - Jupyter notebook, python, and requirements.txt files also need to be updated and included in the PR as well? Or is this just a README.md update?

**Review by ishaghosh27 — DISMISSED** [review]

**Comment by NeoZhangJianyu** [comment]
> @NeoZhangJianyu Do the other files - Jupyter notebook, python, and requirements.txt files also need to be updated and included in the PR as well? Or is this just a README.md update?

Yes, I add new requirements.txt:   AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/requirements.txt

**Review by Marina-Zubova — APPROVED** [review]

**Review by ishaghosh27 — APPROVED** [review]

**Comment by jimmytwei** [comment]
Updated in `release/2024.2_AITools` and `development` branches. Closing this PR out.

**Pull request closed without merging**